ckb/src/main/java/org/nervos/ckb/utils/address/AddressParser.java

```diff
@@ -46,6 +46,9 @@ public static AddressParseResult parse(String address) throws AddressFormatExcep
       }
     }
 
+    if (payload.length() < 66) {
```

Comment:
I also checked the size of `code_hash` when the `Script` is creating.

https://github.com/nervosnetwork/ckb-sdk-ruby/pull/260/files#diff-1717b1cbcb46fd9c2b19c5bf254601e6R15